hub/saml-engine/src/test/java/uk/gov/ida/hub/samlengine/validation/country/EidasAttributeStatementAssertionValidatorTest.java

```diff
@@ -95,7 +106,20 @@ public void shouldInsistOnMandatoryAttributes() {
 
         when(attributeStatement.getAttributes()).thenReturn(List.of(firstName, lastName, dateOfBirth));
 
-        validator.validate(assertion);
+        validator.validate(response, assertion);
+    }
+
+    @Test
+    public void validateEidasAttributesWhenEidasXMLNamespaceURIOnResponse() throws UnmarshallingException, XMLParserException {
+        Security.addProvider(new BouncyCastleProvider());
+        IdaSamlBootstrap.bootstrap();
+        InputStream responseIS = getClass().getClassLoader().getResourceAsStream("response-with-eidas-namespace.xml");
+        InputStream assertionIS = getClass().getClassLoader().getResourceAsStream("assertion-without-eidas-namespace.xml");
```

Comment:
We don't need XML resource files here - XML objects can be mocked. Other tests in this class do it so the approach can be copied in this test too

Reply:
taken out now 👍  although I think using Mockito is perhaps more cognitive overhead than necessary here